PYME/Acquire/Hardware/Camera.py

```diff
@@ -599,7 +600,20 @@ def GetAcquisitionMode(self):
         Returns
         -------
         int
-            One of self.MODE_CONTINUOUS, self.MODE_SINGLE_SHOT
+            One of: (int code: mode name)
+                1: CONTINUOUS
+                    camera is free-running using internal timing
+                0: SINGLE_SHOT
+                    captures single frame on acquisition start
+                2: SOFTWARE_TRIGGER
+                    captures a single frame every time a software trigger is
+                    sent. See `FireSoftwareTrigger`
+                3: HARDWARE_TRIGGER
+                    captures a single frame every time an external hardware
+                    trigger is received by the camera
+                4: HARDWARE_START_TRIGGER
+                    camera waits for an external hardware trigger and enters 
+                    free-running internal timing mode once received

```

Comment:
We make **no** assurances that the integer values will remain consistent, while unlikely, these might change in the future. Calling code should use (and where needed test against) the enum defines (`Camera.MODE_CONTINUOUS` etc) rather than their integer equivalents. This is standard best practice, and our code should reflect this